aks.tf

```diff
@@ -0,0 +1,89 @@
+resource "azurerm_kubernetes_cluster" "aks" {
+  lifecycle {
+    ignore_changes = [
+      default_node_pool[0].node_count
+    ]
+  }
+
+  name                       = local.cluster_name
+  enable_pod_security_policy = false
```

Comment:
why is this disabled? although if we upgrade providers, this disappears. Maybe we shouldn't be using 2.7 but 2.46.1?

Reply:
good question, upgraded